server/src/main/java/com/lunchtime/repository/EventRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.lunchtime.repository;
+
+import com.lunchtime.entity.Event;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Date;
+import java.util.List;
+
+
+public interface EventRepository extends CrudRepository<Event, Long> {
+
+    List<Event> findByDateGreaterThanAndIsActiveTrueOrderByDateAsc(Date date);
```

Comment:
The name of method is to long. Method should make one handle. If they make more that one, you schould make another method. KISS principle.

Reply:
What is an alternative?

Reply:
You can use few simple methods and connect them

Reply:
Nice magic! I didn't know about Query by Example.
In this case your code looks well, but I think you should left method `findAll` and add method `findAllSorted` or `findAllActive`. Because `findAll` can be misleading in your case